### PR TITLE
Security section in Pipelines + a new page - "Removing users in case of a security incident"

### PR DESCRIPTION
**Pull request by karensawrey**
Initially, this PR was adding one new page - "Removing users in case of a security incident", but after some discussions with @plaindocs, we decided that that new page should go into a "Security" section in Pipelines, but there was no "Security" section in Pipelines yet. 

Now, this PR:
- "Security" section in Pipelines
- Adds "Removing users in case of a security incident" page in the Security section
- Moves https://buildkite.com/docs/pipelines/secrets to the Security section 
- Moves https://buildkite.com/docs/pipelines/incoming-webhooks to the Security section 
- Moves https://buildkite.com/docs/pipelines/permissions to the Security section 
- Moves https://buildkite.com/docs/pipelines/audit-log to the Security section 
- Updates the sidebar accordingly

### CODE REVIEW

**Comment by karensawrey** [comment]
Please take a look at this PR, @plaindocs.
I don't understand what are the linter's problems with this PR as it doesn't appear to have trailing whitespace when I inspect the document in the editor. 🤔

**Comment by plaindocs** [comment]
Do you need to escape the `-` in the mail address?

File: pages/agent/v3/removing_users.md.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Within Buildkite's access model, organizations don't own users, so they can't control users' sessions because users represent individuals who may be members of multiple organizations.
+
+Removing a compromised user from your organization immediately protects all the organization's resources from that user. The user will technically still be able to view their personal settings page.
+
+In case of a non-responsive or rogue user, or if multiple accounts are compromised, you can send a list of impacted user IDs to [support@buildkite.com](mailto:support@buildkite.com) and ask the Buildkite support to log out the specific user or all the users out of all sessions.
+
+<section class="Docs__note">
+  <h3>Enterprise users</h3>
+  <p>As a part of the Buildkite’s SLA, Enterprise users have an emergency email available for both operational and security incidents: <a href=”mailto:wake-me-up@buildkite.com”>wake-me-up@buildkite.com</a>.</p>
```
Comment (plaindocs) [inline pages/agent/v3/removing_users.md.erb:23]:
```suggestion
  <p>As a part of the Buildkite’s SLA, Enterprise users have an emergency email available for both operational and security incidents: <a href="mailto:wake-me-up@buildkite.com">wake-me-up@buildkite.com</a>.</p>
```

**Comment by karensawrey** [comment]
@plaindocs please take another look at this one.

**Comment by plaindocs** [comment]
Let's rename the PR and put a small description in now we're making bigger changes.

I like how this looks, although not sure about Webhooks, maybe we need to split out the security content there. I'll dig in tomorrow.

**Comment by karensawrey** [comment]
> Let's rename the PR and put a small description in now we're making bigger changes.
Yep, will do, @plaindocs!
> not sure about Webhooks, maybe we need to split out the security content there. 
The whole incoming webhooks page is basically the security of the incoming webhook page & it says so in the very first paragraph. So maybe let's just rename it, too.

File: app/views/layouts/_sidebar.html.erb [hunk 2]
```diff
@@ -146,8 +144,15 @@
             <%= sidebar_link_to "Controlling Concurrency", 'pipelines/controlling-concurrency' %>
             <%= sidebar_link_to "Branch Configuration", 'pipelines/branch-configuration' %>
             <%= sidebar_link_to "Scheduled Builds", 'pipelines/scheduled-builds' %>
-            <%= sidebar_link_to "Permissions with Teams", 'pipelines/permissions' %>
+
+          </ul>
+          <p class="Docs__nav__section-subheading">Security</p>
+          <ul class="Docs__nav__sub-nav">
+            <%= sidebar_link_to "Secrets", 'pipelines/secrets' %>
```
Comment (plaindocs) [inline app/views/layouts/_sidebar.html.erb:151]:
```suggestion
            <%= sidebar_link_to "Managing Secrets", 'pipelines/secrets' %>
```

File: app/views/layouts/_sidebar.html.erb [hunk 3]
```diff
@@ -146,8 +144,15 @@
...
             <%= sidebar_link_to "Audit Log", 'pipelines/audit-log' %>
+            <%= sidebar_link_to "Permissions with Teams", 'pipelines/permissions' %>
```
Comment (plaindocs) [inline app/views/layouts/_sidebar.html.erb:153]:
```suggestion
            <%= sidebar_link_to "Users and Teams", 'pipelines/permissions' %>
```

File: app/views/layouts/_sidebar.html.erb [hunk 4]
```diff
@@ -146,8 +144,15 @@
...
+            <%= sidebar_link_to "Incoming Webhooks", 'pipelines/incoming-webhooks' %>
```
Comment (plaindocs) [inline app/views/layouts/_sidebar.html.erb:154]:
```suggestion
            <%= sidebar_link_to "Webhooks Security", 'pipelines/incoming-webhooks' %>
```

File: app/views/layouts/_sidebar.html.erb [hunk 5]
```diff
@@ -146,8 +144,15 @@
...
+            <%= sidebar_link_to "Removing Users", 'pipelines/removing-users' %>
```
Comment (plaindocs) [inline app/views/layouts/_sidebar.html.erb:155]: Do you think it makes sense to merge this into Users and Teams? I think it probably does, as long as we're clear which features are *Teams* and which are part of the default.

**Review by plaindocs — COMMENTED** [review]
Yeah, I think if we're making a security section we''ll need some kind of overview of what makes Buildkite so secure by default as well as tweaking the menu links and titles so they make more sense.

File: app/views/layouts/_sidebar.html.erb (hunk 4 above)
Comment (karensawrey) [reply app/views/layouts/_sidebar.html.erb:154]: Mmm... I still really want to keep "Incoming in the title without making it too long 🤔

File: app/views/layouts/_sidebar.html.erb (hunk 5 above)
Comment (karensawrey) [reply app/views/layouts/_sidebar.html.erb:155]: At the moment, I'd actually like to keep them separate - that is, if I got you correctly and you mean that we could merge these 3 pages into 1?

**Review by plaindocs — APPROVED** [review]
This looks great! My only suggestion is to move "Removing Users" into "Users and Teams" as a subsection.

**Comment by karensawrey** [comment]
@plaindocs! Ah, okay, I'm merging these two together & then merging the PR when (if! hahaha) the tests pass. Thanks for the review!

**Pull request merged**